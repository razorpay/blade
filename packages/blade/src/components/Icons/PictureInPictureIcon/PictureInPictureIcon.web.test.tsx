import PictureInPictureIcon from './';
import renderWithTheme from '~utils/testing/renderWithTheme.web';

describe('<PictureInPictureIcon />', () => {
  it('should render PictureInPictureIcon', () => {
    const { container } = renderWithTheme(
      <PictureInPictureIcon color="feedback.icon.neutral.intense" size="large" />,
    );
    expect(container).toMatchSnapshot();
  });
});
