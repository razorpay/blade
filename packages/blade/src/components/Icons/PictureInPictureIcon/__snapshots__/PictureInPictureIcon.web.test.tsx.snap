// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PictureInPictureIcon /> should render PictureInPictureIcon 1`] = `
<div>
  <svg
    aria-hidden="true"
    class="Svgweb__StyledSvg-vcmjs8-0"
    data-blade-component="icon"
    fill="none"
    height="20px"
    viewBox="0 0 24 24"
    width="20px"
  >
    <path
      clip-rule="evenodd"
      d="M4.25 19.75C3.2835 19.75 2.5 18.9665 2.5 18V6C2.5 5.0335 3.2835 4.25 4.25 4.25H20.75C21.7165 4.25 22.5 5.0335 22.5 6V18C22.5 18.9665 21.7165 19.75 20.75 19.75H4.25ZM4.5 17.75V6.25H20.5V11H14C13.5359 11 13.0908 11.1844 12.7626 11.5126C12.4344 11.8408 12.25 12.2859 12.25 12.75V17.75H4.5ZM20.5 13V17.75H14.25V13H20.5Z"
      data-blade-component="svg-path"
      fill="hsla(211, 33%, 21%, 1)"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
